app/uk/gov/hmrc/tradergoodsprofiles/controllers/actions/ValidateHeaderAction.scala

```diff
@@ -70,7 +83,12 @@ class ValidateHeaderAction @Inject() (datetimeService: DateTimeService)(implicit
         case _       => None
       },
       Some(
-        InvalidHeaderErrorResponse(datetimeService.timestamp, "X-Client-ID header is missing").toResult
+        InvalidHeaderErrorResponse(
+          uuidService.uuid,
```

Comment:
correlationId ID here has not meaning and it will only create confusion for the user or user support team as this is used to track the request downstream (to EIS), but in here we haven\t set the request to EIS yet